### ISSUE TITLE
Having issue in @sei-js/core ^3.0.2

### ISSUE THREAD

**Issue by 0xtomoon** [sei-protocol/sei-js#79]
I simply installed this package and tried to interact with RPC url but got this issue.

> PATH/node_modules/@sei-js/core/dist/index.js:58
> Object.assign(self, {
>               ^
> ReferenceError: self is not defined

Here are my code base

- package.json
`"@sei-js/core": "^3.0.2",`
- index.ts
```
import { getCosmWasmClient } from "@sei-js/core";

async function main() { 
  const client = await getCosmWasmClient("https://rpc.atlantic-2.seinetwork.io/");
}

main().catch((error) => {
  console.error("An error occurred:", error);
})
```

**Comment by jolanhe** [comment]
I'm having the same problem.

**Comment by codebycarson** [comment]
We have opted to only polyfill on non-node environments, i.e. when "self" is not defined. This will help to ensure developers can use @sei-js without needing to set up polyfills on their frontend projects.

**Comment by codebycarson** [comment]
This issue has been resolved in versions 3.0.3 

https://github.com/sei-protocol/sei-js/pull/83
